package.json

```diff
@@ -0,0 +1,49 @@
+{
+  "name": "@folio/myprofile",
+  "version": "0.1.0",
+  "description": "My profile",
+  "repository": "folio-org/ui-myprofile",
+  "publishConfig": {
+    "registry": "https://repository.folio.org/repository/npm-folio/"
+  },
+  "license": "Apache-2.0",
+  "engines": {
+    "node": ">=6.0.0"
+  },
+  "stripes": {
+    "type": "settings",
+    "displayName": "ui-myprofile.meta.title",
+    "route": "/myprofile"
+  },
+  "scripts": {
+    "lint": "eslint *.js || true",
+    "test": "echo 'placeholder. no tests implemented'"
+  },
+  "devDependencies": {
+    "@folio/eslint-config-stripes": "^2.0.0",
+    "babel-core": "^6.17.0",
+    "babel-eslint": "^7.0.0",
+    "babel-preset-es2015": "^6.18.0",
+    "babel-preset-react": "^6.16.0",
+    "babel-preset-stage-0": "^6.16.0",
+    "babel-register": "^6.18.0",
+    "eslint": "^4.8.0",
+    "webpack": "1.11.0"
+  },
+  "dependencies": {
+    "@folio/stripes-components": "^2.0.20",
+    "@folio/stripes-core": "^2.9.4",
+    "@folio/react-intl-safe-html": "^1.0.1",
+    "babel-core": "^6.17.0",
+    "lodash": "^4.17.4",
```

Comment:
I don't see a usage of lodash in the code. @VictorSoroka1  do we need it?

Reply:
It is not used now, but it is a kind of standard package of every module. I investigated the first commits of several projects like `ui-organization` or `ui-users` and `lodash` is included in each of them in very first commits. I think we could proceed with that approach too.